task/07-yield-tasks.js

```diff
@@ -126,7 +165,20 @@ function* breadthTraversalTree(root) {
  *   [ 1, 3, 5, ... ], [ -1 ] => [ -1, 1, 3, 5, ...]
  */
 function* mergeSortedSequences(source1, source2) {
-    throw new Error('Not implemented');
+    let s1 = source1(),
+    s2 = source2(),
+    value1 = s1.next().value,
+    value2 = s2.next().value;
+while (true)
```

Comment:
fix indentation.

Reply:
if the loop is not infiinite it's better to put exit condition in the while statement instead of using while(true)